android/app/src/main/res/values/string-arrays.xml

```diff
@@ -50,7 +50,6 @@
   </string-array>
 
   <string-array name="track_length">
-    <item>@string/duration_disabled</item>
```

Comment:
Since on and off was implemented separately so removed it from time selection section

Reply:
There are two buttons one for turning off and turning on and another to set the duration.

If "off" will be there in hour selector too then it will be a duplicate functionality. 

If we implement "off" button in hour selector too then i think there will not be any need of the on/off button because there will be only one click difference.

In ios, only one setting is there which have off button as well in the hour selector